pythonforandroid/recipes/android/__init__.py

```diff
@@ -41,7 +41,7 @@ def prebuild_arch(self, arch):
         is_webview = bootstrap_name == 'webview'
         is_service_only = bootstrap_name == 'service_only'
 
-        if is_sdl2 or is_webview or is_service_only:
+        if is_sdl2 or is_webview or is_service_only or bootstrap_name == 'service_library':
```

Comment:
Prefer adding this to the list of conditions above - or in fact, you could just replace the whole thing with `bootstrap_name in ["sdl2", "sdl2_gradle", "webview", "service_only", "service_library"]`. Even the sdl2_gradle there is probably an obselete name.

Reply:
done